packages/test/src/test-worker-config-heuristics.ts

```diff
@@ -0,0 +1,36 @@
+import test, { ExecutionContext } from 'ava';
+import * as sinon from 'sinon';
+import { addDefaultWorkerOptions } from '@temporalio/worker/lib/worker-options';
+import { GiB } from '@temporalio/worker/lib/utils';
+import type * as V8Type from 'v8';
+import type * as OsType from 'os';
+
+test.serial('maxCachedWorkflows heuristic', (t) => {
```

Comment:
At this point, I am considering removing the unit tests. `maxCachedWorkflows`'s heuristic is once again very simple, so there's little value.

Reply:
👍